otelcol/collector_windows.go

```diff
@@ -141,6 +138,19 @@ func openEventLog(serviceName string) (*eventlog.Log, error) {
 	return elog, nil
 }
 
+func loggingOptionsWithEventLogCore(
+	elog *eventlog.Log,
+	serviceConfig **service.Config,
+	userOptions []zap.Option,
+) []zap.Option {
+	return append(
+		// The event log core must run last, so it needs to be the first option
```

Comment:
Given that I already changed that order in the past (and can't recall if by accident or intentionally) explicit wording seems advisable: 
```suggestion
		// The order below must be preserved - see PR #11051
		// The event log core must run last, so it needs to be the first option
```